lib/install.js

```diff
@@ -289,7 +290,12 @@ Installer.prototype.run = function (cb) {
         log.warn('error', postInstallEr.message)
         log.verbose('error', postInstallEr.stack)
       }
-      cb(installEr || postInstallEr, self.getInstalledModules(), self.idealTree)
+      ensureInPath(function isInPath (isInPathEr, isInPath) {
```

Comment:
I think this test should probably go in `/lib/install/validate-tree` but I'll grant that we're not exactly validating the tree here, this is more about validating your install environment, so I don't hate having it in `install.js` either. But regardless, it shouldn't be calling `log.warn`, instead it should be constructing an error object and pushing it on to `idealTree.warnings`, which will be printed out after the installation summary.